scripts/Docker/debian/clang-cross/13/Dockerfile

```diff
@@ -0,0 +1,64 @@
+# Arguments:
+#   IMAGE_ARCH           - the architecture of the image                    [ amd64 | arm64v8 | riscv64 | loongarch64 ]
+#   DOTNET_SDK_VERSION   - the version of dotnet for the Cake script        [ 9.0 | * ]
+#   TOOLCHAIN_VERSION    - the version of the GCC toolchain                 [ 14 | * ]
+#   TOOLCHAIN_ARCH       - the architecture of the GCC toolchain            [ arm-linux-gnueabihf | aarch64-linux-gnu | riscv64-linux-gnu | loongarch64-linux-gnu ]
+#   TOOLCHAIN_ARCH_SHORT - the short form architecture of the GCC toolchain [ armhf | arm64 | loong64 ]
+#   FONTCONFIG_VERSION   - the exact version of libfontconfig1 to use       [ 2.15.0-1 | * ]
+
+ARG IMAGE_ARCH=amd64
+FROM ${IMAGE_ARCH}/debian:12
```

Comment:
This says debian 12 but the folder number is 13.

But,do we need this file instead of just using the debia 10 one? I suppose CI will tell us :)